google-cloud-firestore/lib/google/cloud/firestore/client.rb

```diff
@@ -185,6 +186,47 @@ def col_group collection_id
         end
         alias collection_group col_group
 
+        ##
+        # Creates a filter object.
+        #
+        # @param [FieldPath, String, Symbol] field A field path to filter
```

Comment:
The argument name & type are in opposite order. It has to be:
```
@param argument [data_type] description
```
Source: https://rubydoc.info/gems/yard/file/docs/GettingStarted.md#documenting-code-with-yard

Reply:
Same applies for all other changes in this PR.

Reply:
Have updated as per the convention for the documentation added. 
We should also consider updating the existing documentation to follow the convention.